include/deal.II/dofs/dof_tools.h

```diff
@@ -2197,6 +2203,37 @@ namespace DoFTools
    std::map<Point<DoFHandlerType::space_dimension>, types::global_dof_index, Comp> &point_to_index_map);
 
   /**
+   * Generate text output readable by gnuplot with point data based on the
+   * given map @p support_points.  For each support point location, a string
+   * label containing a list of all DoFs from the map is generated.  The map
+   * can be generated with a call to map_dofs_to_support_points and is useful
```

Comment:
you'll have to help doxygen find this by using the namespace name of the function, or putting `()` behind it, or both.